### ISSUE TITLE
Latest demo containers don't have linux/amd64 platform

### ISSUE THREAD

**Issue by mboersma** [deislabs/containerd-wasm-shims#159]
This seems like a problem with the v0.9.1 demo containers:

```shell
ctr image pull ghcr.io/deislabs/containerd-wasm-shims/examples/slight-rust-hello:v0.9.0
ghcr.io/deislabs/containerd-wasm-shims/examples/slight-rust-hello:v0.9.0:         resolved       |++++++++++++++++++++++++++++++++++++++| 
manifest-sha256:41d465c956cf1ba8ad2cd424a07a275f7b4531c444004029294b83095530a8ec: done           |++++++++++++++++++++++++++++++++++++++| 
layer-sha256:93c9ffabbf0dfcda9d9cae20da0318ab6b9961976a4a9386b568d9cbd1e67e42:    done           |++++++++++++++++++++++++++++++++++++++| 
layer-sha256:ecefbfcf107e0dc2d00e0627b4ab4cf9a0c9a90f5e37b8fe0c1980340bb526e6:    done           |++++++++++++++++++++++++++++++++++++++| 
config-sha256:801005d9437a75d3cf663bb703775deeb1a037861141134c6d3d0303d434b3ba:   done           |++++++++++++++++++++++++++++++++++++++| 
layer-sha256:fafaac574a1c0c1ca9531b5d8f283de2f6cc11671d371680d0caaed06997a86b:    done           |++++++++++++++++++++++++++++++++++++++| 
elapsed: 1.1 s                                                                    total:  2.0 Ki (1.8 KiB/s)          
                             
unpacking linux/amd64 sha256:41d465c956cf1ba8ad2cd424a07a275f7b4531c444004029294b83095530a8ec...
done: 168.623677ms

ctr image pull ghcr.io/deislabs/containerd-wasm-shims/examples/slight-rust-hello:v0.9.1
ghcr.io/deislabs/containerd-wasm-shims/examples/slight-rust-hello:v0.9.1:      resolved       |++++++++++++++++++++++++++++++++++++++| 
index-sha256:efa2f914708b51fc581b24efdf101dc73e71de70bffaf5cff332190d75d34326: done           |++++++++++++++++++++++++++++++++++++++| 
elapsed: 0.3 s                                                                 total:   0.0 B (0.0 B/s)               
                          
unpacking linux/amd64 sha256:efa2f914708b51fc581b24efdf101dc73e71de70bffaf5cff332190d75d34326...
ctr: no match for platform in manifest sha256:efa2f914708b51fc581b24efdf101dc73e71de70bffaf5cff332190d75d34326: not found
```

Same for the other runtime examples: all the `v0.9.1` tags seem to be broken this way.

**Comment by Mossaka** [comment]
Should be fixed by #160

**Comment by mboersma** [comment]
Just to follow up, #160 did fix things.

I have a PR to update Cluster API images with wasm-containerd-shims v0.9.1 here: kubernetes-sigs/image-builder#1309
